react-redux-app/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
```

Comment:
all files that return JSX must have the extension .jsx Change this in every file pls